ci-operator/config/openshift/contour-operator/openshift-contour-operator-master.yaml

```diff
@@ -0,0 +1,47 @@
+base_images:
+  base:
+    name: "4.9"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+promotion:
+  name: "4.9"
+  namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.9"
+  namespace: ocp
+tests:
+- as: e2e-aws
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws
+- as: test
```

Comment:
Is this the unit test job? If so, we may want to call it `unit` or `test-unit`.

Reply:
Yes unit tests, I'll update the name to test-unit